pkg/quota_management/expiration_date.go

```diff
@@ -0,0 +1,64 @@
+package quota_management
+
+import (
+	"encoding/json"
+	"gopkg.in/yaml.v2"
+	"strings"
+	"time"
+)
+
+const layout = "2006-01-02 -07:00"
```

Comment:
> Is there some reason this layout was specified? 

Yes, I wanted to make the config file as readable as possible and I didn't want the user to specify a time (only a date is needed here)

RFC3339 would require the user to write a date like

2006-01-02T15:04:05Z07:00

I see different minus in that:

1. hard to read
2. hard to write
3. force the user to set a time, while we don't require any time. We don't need that.

Reply:
I understand the motivation :+1: .

Sometimes I prefer adhering to standards at the expense of losing a little bit of convenience.

> force the user to set a time, while we don't require any time. We don't need that.

Isn't parse permissive in that regard, not fully following the specified layout? In `time.Parse` documentation I see the following:
```
Elements omitted from the layout are assumed to be zero or, when zero is impossible, one, so parsing "3:04pm" returns the time corresponding to Jan 1, year 0, 15:04:00 UTC (note that because the year is 0, this time is before the zero Time). Years must be in the range 0000..9999. The day of the week is checked for syntax but it is otherwise ignored. 
```

Reply:
I don't think so. See https://go.dev/play/p/L5mPCCqg54H

Reply:
I see :+1:
Let's go with the proposed layout in the PR then.

I think I understand the comment about omitted elements now: It refers to when defining the layout, if some part is omitted in the layout specification, when the time is parsed the Time value contains that unspecified part in the layout as zero or if impossible one